.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,15 @@ resolves #1235
 ### This pullrequest changes
 
 <!-- Please describe what your pullrequest is changing -->
+
+<cut/>
+
+### Pull Request Readiness Checklist
+
+- [ ] I agree to contribute to the project under OpenCV ([BSD](https://github.com/opencv/opencv/blob/master/LICENSE)) license
+- [ ] The PR is proposed to proper branch ([details](https://github.com/opencv/opencv/wiki/How_to_contribute#before-you-start-contributing-you-should))
```

Comment:
should probably be changed to:

The patch is submitted to the proper branch, 3.4 (bug fixes that are applicable both to 3.4 and master) or master (new functionality, bug fixes for the new functionality).